Content.Server/Punpun/PunpunSystem.cs

```diff
@@ -0,0 +1,109 @@
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.Inventory;
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Components;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Punpun;
+
+public sealed class PunpunSystem : EntitySystem
+{
+    [Dependency] private readonly InventorySystem _inventory = default!;
+    [Dependency] private readonly ServerMetaDataSystem _meta = default!;
+
+    private (int, string, string) _punpunData = (0, string.Empty, string.Empty);
+
+    // All the roman numerals we'll need to display
+    private readonly Dictionary<int, string> _numerals = new()
+    {
+        { 0, "I" },
+        { 1, "II" },
+        { 2, "III" },
+        { 3, "IV" },
+        { 4, "V" },
+        { 5, "VI" },
+        { 6, "VII" },
+        { 7, "VIII" },
+        { 8, "IX" },
+        { 9, "X" },
+        { 10, "XI" },
+        { 11, "XII" },
+        { 12, "XIII" },
+        { 13, "XIV" }
+    };
```

Comment:
```suggestion
    // All the roman numerals we'll need to display
    private readonly Dictionary<int, string> _numerals = new()
    {
        { 0, "I" },
        { 1, "II" },
        { 2, "III" },
        { 3, "IV" },
        { 4, "V" },
        { 5, "VI" },
        { 6, "VII" },
        { 7, "VIII" },
        { 8, "IX" },
        { 9, "X" },
        { 10, "XI" },
        { 11, "XII" },
        { 12, "XIII" },
        { 13, "XIV" }
    };
```
Do you think we can port in a roman numeral constructor here? I have a specific reason for asking.

Reply:
I was thinking about that, I can look for something